src/constants.ts

```diff
@@ -528,22 +528,29 @@ function getFwVersionConst (v) {
 // eslint-disable-next-line no-control-regex
 const ASCII_REGEX = /^[\x00-\x7F]+$/;
 
+const EXTERNAL_NETWORKS_BY_CHAIN_ID_URL =
+  'https://gridplus.github.io/chains/chains.json';
```

Comment:
Interesting... how did you map to this URL?

Reply:
The `chains` repo deploys whatever is in its `/data` folder to github pages with [this workflow.](https://github.com/GridPlus/chains/blob/main/.github/workflows/build-deploy.yml) So the `chains.json` file is deployed there and we can grab it here.

Reply:
Commits to `main` in that repo will automatically deploy an update.

Reply:
😮